Updates to README

mbbroberg opened the pull request:
Redesigning the README based on feedback.

mbbroberg committed:
Reorganizing content

* Think it's clearer to have the schedule and this year's teams closer to the top and clarifying the role of the challenge

rnelson0 commented on README.md:
```diff
@@ -1,33 +1,75 @@
+- **Wednesday 11/30** - Git'ing Ops in shape for DevOps with [Taylor Riggan](https://twitter.com/triggan)
+- **Monday 12/5** - Lessons learned from writing the DevOps Handbook with [Gene Kim](https://twitter.com/realgenekim)
+- **Wednesday 12/7** at **8P CST/9P EST** - Forking public work, committing to it, and managing the PR process with [Safia Abdalla](https://twitter.com/captainsafia)
+- **Wednesday 12/14** - Curating a group of people that build tools together with [Matthew Brender](https://twitter.com/mjbrender:)
```
Errant colon in your twitter link

mjbrender commented:
This is out of date due to the latest merge, which is no big deal. I'll update - thanks for the review @rnelson0. cc @jfrappier

mbbroberg committed:
Correcting schedule based on master

* Resolving merge conflict from 1b93c0ca3b768b7c6dd99044141ea8ca0d0b390f

mbbroberg committed:
Removing time specified for 12/7

Because it makes the section less readable on GH. Since that slot has passed, no harm to removing it for aesthetic reasons

rnelson0 reviewed (approved):
Assuming this is no longer a WIP I approve. Merge away! If it's still in progress, ignore this :)

mjbrender commented:
All set now - thanks for the review @rnelson0!
